src/main/createWindow.js

```diff
@@ -21,7 +21,7 @@ const createWindow = (pathname, options = {}) => {
     },
     useContentSize: true,
     show: false,
-    frame: false,
+    frame: (process.platform === 'linux' ? false : true),
```

Comment:
frame: process.platform === 'linux',

So don't need to use condition ? expr1 : expr2'